san/get.py

```diff
@@ -9,9 +9,17 @@
     'ohlcv': 'get_ohlcv'
 }
 
+DEPRECATED_QUERIES = {
+    'mvrv_ratio': 'mvrv_usd',
+    'nvt_ratio': 'nvt',
+    'realized_value': 'realized_value_usd',
+    'token_circulation': 'circulation'
+}
```

Comment:
There are missing ones like: `burn_rate/token_age_consumed -> age_destroyed`

Reply:
Didn't know about that one, fix is on the way